charts/victoria-logs-cluster/_index.md.gotmpl

```diff
@@ -78,17 +78,19 @@ In the example above, two `vlstorage` instances are deployed, each with a 30-day
 
 To access the data within the cluster, use the vmauth address: `http://vlc-victoria-logs-cluster-vmauth.logging.svc.cluster.local:8427`.
 You can use this address as Data Source URL in Grafana.
-The vmui interface is available at: `http://vlc-victoria-logs-cluster-vmauth.logging.svc.cluster.local:8427/select/vmui/`.
+
+Standalone VMUI interface is available at: `http://vlc-victoria-logs-cluster-vmauth.logging.svc.cluster.local:8427/select/vmui/`.
+VictoriaLogs can ingest logs from various sources, see [ingestion](https://docs.victoriametrics.com/victorialogs/#ingestion) documentation for more details.
```

Comment:
```suggestion
VictoriaLogs can accept logs at `http://vlc-victoria-logs-cluster-vmauth.logging.svc.cluster.local:8427/insert/*` according to [these docs](https://docs.victoriametrics.com/victorialogs/data-ingestion/).
```

Please verify whether the referred url - `http://vlc-victoria-logs-cluster-vmauth.logging.svc.cluster.local:8427/insert/*` is correct for data ingestion for VictoriaLogs deployed with the given Helm chart.